internal/pkg/skuba/deployments/ssh/kubernetes.go

```diff
@@ -77,7 +78,12 @@ func kubernetesInstallNodePattern(t *Target, data interface{}) error {
 	return err
 }
 
-func kubernetesRestartServices(t *Target, data interface{}) error {
-	_, _, err := t.ssh("systemctl", "restart", "crio", "kubelet")
+func kubernetesStartKubelet(t *Target, data interface{}) error {
+	_, _, err := t.ssh("systemctl", "enable", "--now", "kubelet")
+	return err
+}
+
+func kubernetesStopKubelet(t *Target, data interface{}) error {
+	_, _, err := t.ssh("systemctl", "stop", "kubelet")
```

Comment:
Maybe it's better put this to `internal/pkg/skuba/deployments/ssh/kubelet.go`.

Reply:
You are absolutely right! Let's first see if we need this PR at all (if we go with cri-o 1.16 we don't need it) -- if we do, will update, forgot about `ssh/kubelet.go` completely :)